tasks/common_packages.yml

```diff
@@ -34,3 +34,11 @@
       - locales
       - nfs-client
       - net-tools
+      - tree
+      - tmux
+      - ncdu
+      - iostat
+      - dialog
+      - nmap
+      - curl
+      - wget
```

Comment:
Why when you already have curl? It's like installing grub 2 and grub 1 on the same system - just doesn't make any sense.